resources/authgear/templates/en/web/__error.html

```diff
@@ -169,18 +174,23 @@
     <li>{{ template "error-developer-reauthentication" }}</li>
     {{ else }}
     <li>{{ .Error.message }}</li>
-    {{ end }} {{ else if eq .Error.reason "InvalidVerificationCode" }}
-    <li>
-      {{ template "error-verification-code-invalid" }}
-      <a
-        href="#"
-        data-transfer-click-click-value="#verify-identity-resend-button"
-        data-controller="transfer-click"
-        data-action="transfer-click#click"
-      >
-        {{ template "error-verification-code-invalid-click-to-resend" }}
-      </a>
-    </li>
+    {{ end }}
+    {{ else if eq .Error.reason "InvalidVerificationCode" }}
+      {{ if eq $info.Channel "whatsapp" }}
+        <li>{{ template "error-invalid-whatsapp-otp" }}</li>
+      {{ else }}
+        <li>
+          {{ template "error-verification-code-invalid" }}
```

Comment:
I think we should keep using this error message, as it provides a way to resend.